File: src/brs/services/ATService.java
package brs.services;

import brs.at.AT;

import java.util.Collection;
import java.util.List;

public interface ATService {

  Collection<Long> getAllATIds();

  List<Long> getATsIssuedBy(Long accountId);

  AT getAT(Long atId);
  
  AT getAT(Long atId, int height);
}

File: src/main/kotlin/brs/services/ATService.kt
package brs.services

import brs.at.AT

interface ATService {
    /**
     * TODO
     */
    fun getAllATIds(): Collection<Long>

    /**
     * TODO
     */
    fun getATsIssuedBy(accountId: Long): List<Long>

    /**
     * TODO
     */
    fun getAT(id: Long): AT?
}
